app/controllers/profiles_controller.rb

```diff
@@ -40,7 +40,7 @@ def changed_email?
 
   def set_mail_changes
     flash[:notice] = I18n.t('flash.notice.profile_email_updated')
-    force_send_confirmation_email if @user.email.blank?
+    force_send_confirmation_email
```

Comment:
@tauil I made this change because the tests were not passing after devise update.
IMHO, there is reason to confirm emails just from users without it. If I want to change my current email for a new one, makes sense I also confirm this one.